### PR TITLE
Add per-repo dictionary overrides

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
This adds per-repo overrides.  Depends on #3 

## Pull Request Checklist
Thank you for submitting a contribution to `goat`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
- [X] Rebase your branch against the latest commit of the target branch, which likely should be `main`
- [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

**Comment by JonZeolla** [comment]
Blocked on https://github.com/streetsidesoftware/cspell/issues/924

File: etc/cspell.config.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+if (process.env.GITHUB_WORKSPACE) {
```
Comment (Jason3S) [inline etc/cspell.config.js]: How about?:
```
const BASEPATH = process.env.GITHUB_WORKSPACE || '.'
```

File: .github/etc/dictionary.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+capricornis
+crispus
```
Comment (tpears) [inline .github/etc/dictionary.txt:2]: A local execution of `pipenv run invoke goat` returns the following cspell error even with the `etc/seiso_global_dictionary.txt` updated to include these terms

>{"timestamp": "2021-02-10 09:58:08,281", "namespace": "seiso.goat", "loglevel": "ERROR", "message": "Received a baaaaaaaaah-d status code from docker (1); additional details: /usr/bin/dockerfile_lint -> /usr/lib/node_modules/dockerfile_lint/bin/dockerfile_lint  + dockerfile_lint@0.3.4  added 59 packages from 83 contributors in 5.237s    # Analyzing ./Dockerfile    Check passed!  /usr/bin/cspell -> /usr/lib/node_modules/cspell/bin.js  + cspell@5.2.4  added 124 packages from 60 contributors in 8.143s  /tmp/lint/README.md:38:1 - Unknown word (crispus)  1/1 ./README.md 2194.16ms  CSpell: Files checked: 1, Issues found: 1 in 1 files"}

Comment (tpears) [reply .github/etc/dictionary.txt:2]: Was using Goat 0.2.0 to test Goat 0.2.1. Had to build the image from this branch first... duh!

New error now though:

>{"timestamp": "2021-02-10 10:08:14,117", "namespace": "seiso.goat", "loglevel": "INFO", "message": "Baaaaaaaaaaah! (Running the goat)"}
{"timestamp": "2021-02-10 10:09:26,627", "namespace": "seiso.goat", "loglevel": "INFO", "message": "Linting tests passed"}
{"timestamp": "2021-02-10 10:13:52,718", "namespace": "seiso.goat", "loglevel": "ERROR", "message": "Received a baaaaaaaaah-d status code from docker (1); additional details: 2021-02-10T15:13:52.293Z	FATAL	error in image scan: failed analysis: analyze error: timeout: context deadline exceeded"}

File: etc/cspell.config.js (hunk 1 above)
Comment (JonZeolla) [reply etc/cspell.config.js]: See https://github.com/SeisoLLC/goat/pull/4/commits/0ee502a0bfd52e1eb46ab9e9375f5900c28cf028; I replaced it with a ternary

File: .github/etc/dictionary.txt (hunk 2 above)
Comment (tpears) [reply .github/etc/dictionary.txt:2]: Timeout continues to occur in ~5m even after increasing the timeout specified in`task.py` as follows

>CLIENT = docker.from_env(timeout=600)

**Review by jimmio — APPROVED** [review]
LGTM

**Review by tpears — APPROVED** [review]
Local testing failure not a blocker. PR approved

File: .github/etc/dictionary.txt (hunk 2 above)
Comment (JonZeolla) [reply .github/etc/dictionary.txt:2]: I have seen this intermittently and don't have a workaround yet; on backlog.

**Review by jimmio — APPROVED** [review]

**Pull request merged**